src/components/FAQSection/QuestionFAQ/QuestionFAQ.styles.tsx

```diff
@@ -0,0 +1,62 @@
+import { Icon } from "@/components/Icon/Icon";
+import styled from "styled-components";
+
+export const SubSectionFAQ = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  padding-left: 0.5rem;
+  transition: 100ms;
+  margin-top: 0;
+  margin-left: 0.5rem;
+`;
```

Comment:
Sugested a different tag, the <p> is more meaningful.

```suggestion
export const SubSectionFAQ = styled.p`
  display: flex;
  align-items: center;
  gap: 0.5rem;
  transition: 100ms;
  padding: 0 1rem;
`;
```